source/chapters/user_interface.rst

```diff
@@ -707,38 +707,123 @@ Channel Faders and Level Meters
    :figclass: pretty-figures
 
-   The section expansion buttons on top
+   EQ Controls
 
-  .. figure:: ../_static/2.2/expansion-buttons-skin-settings.png
-    :align: center
-    :width: 245px
-    :figwidth: 90%
-    :alt: The section expansion buttons
-    :figclass: pretty-figures
+**Gain Knob**
+  The gain knob allows you to adjust the gain applied to the deck.
+  Use this to compensate for the differences in recording levels between tracks.
+  In general, you should adjust this knob so that the track's :term:`level meter`
+  stays around the top of the green region with the loudest parts of the track
+  (the transients) briefly going into the yellow region. **Do NOT turn the
+  Gain Knob up so much that the level meter is in the red region. At this point
+  the track is clipping, which sounds bad and could damage equipment.**
 
-    The section expansion options under the Skin settings on the right.
+  .. note:: By default, Mixxx automatically applies an additional ReplayGain so
+            tracks have approximately equal loudness at unity gain. Your tracks
+            must be :ref:`analyzed <getting-started-analyze-library>` to take
+            advantage of this feature. When an unanalyzed track is
+            loaded, Mixxx calculates its ReplayGain value, but will not apply a
+            newly calculated ReplayGain value to a track after it has
+            already started playing (to avoid a sudden change in the gain of a
+            playing track).
 
-On top, and on the right side of the user interface are the section expansion buttons and options.
+  .. seealso:: For an explanation of why you should set your gains this way, see
+               :ref:`Setting your levels properly (gain staging) <djing-gain-staging>`.
 
-If you click on either :guilabel:`EFFECTS`, :guilabel:`SAMPLERS` or :guilabel:`MICS & AUX` then you will enable control sections for interacting with:
+**EQ Knobs**
+  The low, mid, and high EQ knobs allow you to change the filters of the audio.
+  This allows you to selectively reduce or boost certain frequency ranges of
+  the audio.
 
-* :ref:`interface-effects`
-* :ref:`interface-sampler`
-* :ref:`interface-mic`
+  .. warning:: Be careful not to raise the EQs so much that the signal clips,
+               indicated by a :term:`level meter` being in the red region. See
+               :ref:`Setting your levels properly (gain staging) <djing-gain-staging>`
+               for a more detailed explanation.
 
-You can also use the specific :ref:`appendix-shortcuts`.
+**Kill Switches**
+  The small boxes next to each EQ knob are called :term:`kills <kill switch>`.
+  Hold these buttons to fully remove that frequency range. Short click on the
+  buttons for latching. When in Latch mode, click again to restore the frequency
+  range. If the Kill switches do not work as expected, check the high/low shelf
+  EQ settings in the preferences.
+
+.. seealso:: You can customize the EQ settings in
```

Comment:
Here we need to describe the Quick Effect, as well:

```suggestion

**Quick Effect knob**
  This knob let's you control the Meta knob of the deck's Quick Effect. 'Filter' is the default.
  Right-click this knob to center it. This is equivalent to 'no audible effect' for the built-in
  filter effects only (Filter, Moog Ladder 4 Filter).
  The button next to the knob toggles the effect. It provides the same latching capabilities
  like the EQ Kill switches.

.. seealso:: You can customize the EQ and Quick Effect settings in
```

Reply:
Hello @ronso0 I have done this!